.buildkite/pipeline.yaml

```diff
@@ -55,6 +59,10 @@ steps:
     artifact_paths: _out_/**/*
     agents:
       os: mac
+    # TODO(jez) We should only retry for specific error.
+    # The one we were trying to fix was `'foo.h': file not found`
+    retry:
+     automatic: true
```

Comment:
See docs here:

https://buildkite.com/docs/pipelines/command-step

default settings are to retry twice.